store/store.go

```diff
@@ -1391,3 +1409,12 @@ func (s *Store) PaymentMethods(user *auth.UserState) (*PaymentInformation, error
 		return nil, fmt.Errorf("cannot get payment methods: unexpected HTTP code %d%s", resp.StatusCode, details)
 	}
 }
+
+func (s *Store) listRefreshFields() []string {
+	fields := append([]string{}, s.detailFields...)
```

Comment:
Actually - I made this change but then the failing test reminded me why I had it the way I had :). Without the copy, fields is a reference to the same slice s.detailFields, and so we end up setting s.detailFields with the appended slice. Here: http://paste.ubuntu.com/23202537/

So I've pushed without this change.